Server/src/bgp/linkstate/MPLinkStateAttr.cpp

```diff
@@ -352,9 +352,8 @@ namespace bgp_msg {
                         val_ss << ", " << value_16bit;
                 }
 
-                memcpy(parsed_data->ls_attrs[ATTR_NODE_MT_ID].data(), val_ss.str().data(), val_ss.str().length());
+                strncpy((char *)parsed_data->ls_attrs[ATTR_NODE_MT_ID].data(), val_ss.str().data(), val_ss.str().length()+1);
```

Comment:
Should be okay 99.9% of the time, but could be an issue due to the size of val_ss being larger than ls_attrs.data.    Alternatively to using +1 on len for null term fill(0) could be used to zero the data buffer.